bothub/api/v1/views.py

```diff
@@ -636,13 +637,20 @@ class RepositoryExampleViewSet(
 
     delete:
     Delete repository example.
+
+    update:
+    Update repository example.
+
     """
     queryset = RepositoryExample.objects
     serializer_class = RepositoryExampleSerializer
     permission_classes = [
         RepositoryExamplePermission,
     ]
 
+    def perform_update(self, serializer):
```

Comment:
The method "perform_update" is necessary?

Reply:
is not necessary, i will remove @edudouglas